File: FundTrack.WebUI/ClientApp/app/view-models/concrete/finance/donate/donate-account.view-model.ts
export class DonateAccountViewModel{
    name: string;
    description: string;
    bankAccountId: number;
    merchantId: number;
    merchantPassword: string;
    targetId: number;
    target: string;
}

export class OrganizationDonateAccountsViewModel {
    organizationId: number;
    orgName: string;
    accounts: Array<DonateAccountViewModel>
}

File: FundTrack.Infrastructure/ViewModel/FinanceViewModels/DonateViewModels/DonateAccountViewModel.cs
namespace FundTrack.Infrastructure.ViewModel.FinanceViewModels.DonateViewModels
{
    public class DonateAccountViewModel
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public int BankAccountId { get; set; }
        public int? MerchantId { get; set; }
        public string MerchantPassword { get; set; }
        public int? TargetId { get; set; }
        public string Target { get; set; }
    }
}
